client/my-sites/posts/main.jsx

```diff
@@ -82,7 +82,11 @@ class PostsMain extends React.Component {
 					brandFont
 					className="posts__page-heading"
 					headerText={ translate( 'Posts' ) }
-					subHeaderText={ translate( 'Create, edit, and manage the posts on your site.' ) }
+					subHeaderText={
+						siteId
+							? translate( 'Create, edit, and manage the posts on your site.' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.